libs/version/src/version.cpp

```diff
@@ -199,15 +200,17 @@ namespace hpx {
         strm << "\n";
 
         // print module configurations
-        module_config_strings const* ms = hpx::config_strings_modules;
-        while (ms->name)
+        auto configs = hpx::config_registry::get_module_configs();
```

Comment:
Should this be `auto const& configs = …` instead?

Reply:
Yep, thanks!